cfgov/v1/util/util.py

```diff
@@ -68,6 +68,9 @@ def get_secondary_nav_items(request, current_page):
     else:
         page = get_appropriate_page_version(request, current_page)
 
+    if not page:
```

Comment:
Total interloper so ignore if this is irrelevant: just recently (seems to be an impact of the latest Wagtail update) we regained the ability to preview new pages before they had been shared/saved. For awhile that was a minor annoyance.